config/F32.config

```diff
@@ -88,7 +88,7 @@ process {
     }
     withName: run_sump_MuSE {
         cpus = 12
-        memory = 48.GB
+        memory = 18.GB
```

Comment:
It may be worth checking the GIAB runs to see how much memory was used by those samples to potentially guide the retry setting if the initial allocation is being reduced this much

Reply:
Strangely, the [results from running the 2.5/1.1 TB BAMs](https://github.com/uclahs-cds/pipeline-call-sSNV/issues/288) indicate only 3GB (according to the trace file).
Two other BAM pairs (312 GB, 328 GB with 369 GB Normal) used 9.1 GB and 32.4 GB.  The later would work on retry on F72.

Reply:
Interesting, it may be some odd resource accounting